automated install/basic-install.sh

```diff
@@ -176,29 +192,30 @@ is_repo() {
 make_repo() {
   local directory="${1}"
   local remoteRepo="${2}"
-
-  echo -n ":::    Cloning ${remoteRepo} into ${directory}..."
+  str="Clone ${remoteRepo} into ${directory}"
```

Comment:
I'm seeing a couple of instances where the variable name `str` is used. I'm probably being *wayyy* too pedantic, but for readability sake, would it be better to use something more specific like `clone_str`?